CMakeLists.txt

```diff
@@ -66,7 +66,7 @@ endif()
 if(CSFML_BUILD_NETWORK)
     list(PREPEND SFML_MODULES "Network")
 endif()
-find_package(SFML 3.0.0 COMPONENTS ${SFML_MODULES} REQUIRED)
+find_package(SFML 3.0.2 COMPONENTS ${SFML_MODULES} REQUIRED)
```

Comment:
Does CSFML need anything from 3.0.2 in particular? My understanding is that CSFML should work with any version of SFML 3.

Reply:
I guess we could stick to 3.0 or does 3.0.0 also allow to find newer patch versions? (Didn't I ask this in the past already? 😄)

Reply:
I think `3.0` makes the most sense, since we do want to target SFML 3.0 and don't care whether it's 3.0.0 or 3.0.2